modular_skyrat/modules/opposing_force/code/opposing_force_datum.dm

```diff
@@ -436,10 +436,19 @@
 	if(ishuman(target))
 		var/mob/living/carbon/human/human = target
 		human.put_in_hands(spawned_box)
+	if(target.mind.has_antag_datum(/datum/antagonist/traitor))
+		var/datum/component/uplink/uplink = target.mind.find_syndicate_uplink(TRUE)
+		var/obj/item/stack/telecrystals = create_tc()
+		uplink.load_tc(user, telecrystals)
 
 	add_log(user.ckey, "Issued gear")
 	send_system_message("[user ? get_admin_ckey(user) : "The OPFOR subsystem"] has issued all approved equipment")
-	equipment_issued = TRUE
+
+/datum/opposing_force/proc/create_tc()
```

Comment:
needs documentation

Reply:
it's a magic number right now but ima attach it to something